Tasks/AzurePowerShellV4/TryMakingModuleAvailable.ps1

```diff
@@ -0,0 +1,78 @@
+[CmdletBinding()]
+param (
+    [string]
+    $targetVersion,
+
+    [string] [Parameter(Mandatory = $true)] [ValidateSet("Windows", "Linux", "Mac")]
+    $platform
+)
+
+try {
+    $isWin = $platform -eq "Windows"
+    if ($isWin) {
+        . "$PSScriptRoot\Utility.ps1"
+        $isHostedAgent = Test-IsHostedAgentPathPresent
+    } else {
+        . "$PSScriptRoot/Utility.ps1"
+        $isHostedAgent = Test-IsHostedAgentPathPresentLinux
+    }
+
+    if (!$isHostedAgent) {
+        $moduleSource = "Others"
+        return
```

Comment:
there are multiple returns from this file, code maintenance will be hard in long term

Reply:
I would argue otherwise since having multiple early return provides more clear code than having multiple nested if statements.
Please let me know in case you think otherwise.